config/samples/python_example_scenario.json

```
@@ -0,0 +1,65 @@
+{
+  "scenarios": [
+    {
+      "benchmark_seconds": 30,
+      "client_config": {
+        "async_client_threads": 1,
+        "channel_args": [
+          {
+            "name": "grpc.optimization_target",
+            "str_value": "latency"
+          }
+        ],
+        "client_channels": 1,
+        "client_processes": 0,
+        "client_type": "SYNC_CLIENT",
+        "histogram_params": {
+          "max_possible": 60000000000.0,
+          "resolution": 0.01
+        },
+        "load_params": {
+          "closed_loop": {}
+        },
+        "outstanding_rpcs_per_channel": 1,
+        "payload_config": {
+          "bytebuf_params": {
+            "req_size": 0,
+            "resp_size": 0
+          }
+        },
+        "rpc_type": "STREAMING",
+        "security_params": {
+          "server_host_override": "foo.test.google.fr",
+          "use_test_ca": true
+        },
+        "threads_per_cq": 0
+      },
+      "name": "python_example_scenario",
```

Why don't we move the name to the top, so it is quickly visible?

```suggestion
      "name": "python_example_scenario",
      "benchmark_seconds": 30,
      "client_config": {
        "async_client_threads": 1,
        "channel_args": [
          {
            "name": "grpc.optimization_target",
            "str_value": "latency"
          }
        ],
        "client_channels": 1,
        "client_processes": 0,
        "client_type": "SYNC_CLIENT",
        "histogram_params": {
          "max_possible": 60000000000.0,
          "resolution": 0.01
        },
        "load_params": {
          "closed_loop": {}
        },
        "outstanding_rpcs_per_channel": 1,
        "payload_config": {
          "bytebuf_params": {
            "req_size": 0,
            "resp_size": 0
          }
        },
        "rpc_type": "STREAMING",
        "security_params": {
          "server_host_override": "foo.test.google.fr",
          "use_test_ca": true
        },
        "threads_per_cq": 0
      },
```

---

I kept it as it was generated just to make sure that if others interested in generate this scenario themselves, they will have exactly the same result as we have here, but I did get the idea that the original way makes it hard to people to see the scenario name, I agree the change. :)